tests/mq/plugins/test_broFixup.py

```diff
@@ -0,0 +1,1697 @@
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(__file__), "../../../lib"))
+from utilities.toUTC import toUTC
+
+import mock
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "../../../mq/plugins"))
+from broFixup import message
+
+
+class TestBroFixup(object):
+    def setup(self):
+        self.plugin = message()
+        self.metadata = {
+            'doc_type': 'nsm',
+            'index': 'events'
+        }
+
+    def test_notbro_log(self):
+        metadata = {
+            'doc_type': 'event',
+            'index': 'events'
+        }
+        event = {
+            'key1': 'bro'
+        }
+        
+        result, metadata = self.plugin.onMessage(event, metadata)
+        assert result
+        assert metadata['doc_type'] is not 'nsm'
+
+    def test_notbro_log2(self):
+        metadata = {
+            'doc_type': 'event',
+            'index': 'events'
+        }
+        event = {
+           'bro': 'value1'
+        }
+       
+        result, metadata = self.plugin.onMessage(event, metadata)
+        assert result
```

Comment:
Same comment, what are you asserting here?

Reply:
Changed to compare event on input with event on the output. Should be the same, those tests are to make sure non-bro messages do not match.